perception/object_recognition/detection/object_merger/src/object_association_merger/node.cpp

```diff
@@ -67,28 +64,25 @@ void ObjectAssociationMergerNode::objectsCallback(
   Eigen::MatrixXd score_matrix =
     data_association_.calcScoreMatrix(*input_object1_msg, *input_object0_msg);
   data_association_.assign(score_matrix, direct_assignment, reverse_assignment);
-  for (size_t object0_idx = 0; object0_idx < input_object0_msg->feature_objects.size();
-       ++object0_idx) {
+  for (size_t object0_idx = 0; object0_idx < input_object0_msg->objects.size(); ++object0_idx) {
     if (direct_assignment.find(object0_idx) != direct_assignment.end()) {  // found
       // The one with the higher score will be hired.
       if (
-        input_object1_msg->feature_objects.at(direct_assignment.at(object0_idx))
-          .object.semantic.confidence <
-        input_object0_msg->feature_objects.at(object0_idx).object.semantic.confidence) {
-        output_msg.feature_objects.push_back(input_object0_msg->feature_objects.at(object0_idx));
+        input_object1_msg->objects.at(direct_assignment.at(object0_idx)).existence_probability <
+        input_object0_msg->objects.at(object0_idx).existence_probability) {
```

Comment:
@yukkysaito @yukke42 please check the following modification :pray: 

- confidence -> existence_probability

Reply:
LGTM